src/index.spec.ts

```diff
@@ -1,16 +1,26 @@
 import {TestOptions} from './options.spec';
+import {getComponentSpec} from './components/get-component.spec';
+import {getComponentsSpec} from './components/get-components.spec';
 import {getCustomersSpec} from './customers/get-customers.spec';
 import {getSubscriptionsSpec} from './subscriptions/get-subscriptions.spec';
 
+const productFamilyId = process.env.TEST_CHARGIFY_PRODUCT_FAMILY_ID && parseInt(process.env.TEST_CHARGIFY_PRODUCT_FAMILY_ID) || 100000;
```

Comment:
wouldn't this eval to either  true or 100000

Reply:
Not in JavaScript https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators

Still double-checked it:
<img width="204" alt="screen shot 2019-01-30 at 10 30 50 am" src="https://user-images.githubusercontent.com/7393416/52003715-295bc180-247a-11e9-8db3-8b2192c8daa2.png">